src/bin/travis/resources/centaur.wdl

```diff
@@ -21,7 +33,7 @@ task centaur {
     }
 
     runtime {
-        docker: "geoffjentry/centaur-cromwell:latest" 
+        docker: "us.gcr.io/broad-dsde-cromwell-dev/centaur:latest" 
```

Comment:
@mcovarr sadly no. That's referring to all the encrypted secret stuff which sits on my laptop, which I currently don't have access to :)